docs/book/driver-deployment/deploying_csi_with_zones.md

```diff
@@ -0,0 +1,315 @@
+
+   [VirtualCenter "vCenter Server IP address"]
+   user = "user"
+   password = "password"
+   port = "443"
+   datacenters = "datacenter"
+
+   [Network]
+   public-network = "VM Network"
+
+   [Labels]
+   region = k8s-region
+   zone = k8s-zone
+   ```
+
+   ```bash
+   cd /etc/kubernetes
+   kubectl create configmap cloud-config --from-file=vsphere.conf --namespace=kube-system
+
+   kubectl apply -f https://raw.githubusercontent.com/kubernetes/cloud-provider-vsphere/master/manifests/controller-manager/cloud-controller-manager-roles.yaml
+   kubectl apply -f https://raw.githubusercontent.com/kubernetes/cloud-provider-vsphere/master/manifests/controller-manager/cloud-controller-manager-role-bindings.yaml
+   kubectl apply -f https://github.com/kubernetes/cloud-provider-vsphere/raw/master/manifests/controller-manager/vsphere-cloud-controller-manager-ds.yaml
+   ```
+
+2. Verify that your CCM installation is successful.
+
+   After installation, labels `failure-domain.beta.kubernetes.io/region` and `failure-domain.beta.kubernetes.io/zone` are applied to all nodes.
+
+   ```bash
+   kubectl get nodes -L failure-domain.beta.kubernetes.io/zone -L failure-domain.beta.kubernetes.io/region
+   NAME         STATUS   ROLES    AGE   VERSION   ZONE     REGION
+   k8s-master   Ready    master   32m   v1.14.2   zone-a   region-1
+   k8s-node1    Ready    <none>   18m   v1.14.2   zone-a   region-1
+   k8s-node2    Ready    <none>   18m   v1.14.2   zone-b   region-1
+   k8s-node3    Ready    <none>   18m   v1.14.2   zone-b   region-1
+   k8s-node4    Ready    <none>   18m   v1.14.2   zone-c   region-1
+   k8s-node5    Ready    <none>   18m   v1.14.2   zone-c   region-1
+   ```
+
+3. Install the CSI driver.
+
+   Make sure `external-provisioner` is deployed with the arguments `--feature-gates=Topology=true`.
+
+   In the credential secret file, add entries for region and zone.
+
+   ```bash
+   [Labels]
+   region = k8s-region
+   zone = k8s-zone
+   ```
+
+4. Verify that your CSI driver installation is successful.
+
+   ```bash
+   kubectl get csinodes -o jsonpath='{range .items[*]}{.metadata.name} {.spec}{"\n"}{end}'
+   k8s-node1 map[drivers:[map[name:csi.vsphere.vmware.com nodeID:k8s-node1 topologyKeys:[failure-domain.beta.kubernetes.io/region failure-domain.beta.kubernetes.io/zone]]]]
+   k8s-node2 map[drivers:[map[name:csi.vsphere.vmware.com nodeID:k8s-node2 topologyKeys:[failure-domain.beta.kubernetes.io/region failure-domain.beta.kubernetes.io/zone]]]]
+   k8s-node3 map[drivers:[map[name:csi.vsphere.vmware.com nodeID:k8s-node3 topologyKeys:[failure-domain.beta.kubernetes.io/region failure-domain.beta.kubernetes.io/zone]]]]
+   k8s-node4 map[drivers:[map[name:csi.vsphere.vmware.com nodeID:k8s-node4 topologyKeys:[failure-domain.beta.kubernetes.io/region failure-domain.beta.kubernetes.io/zone]]]]
+   k8s-node5 map[drivers:[map[name:csi.vsphere.vmware.com nodeID:
+   ```
+
+#### Deploy Workloads Using Zones <a id="deploy_workload_using_zones"></a>
```

Comment:
vSphere CSI Driver - Volume Topology talks about how to use this feature. So, this entire section can be removed since we are just talking about CSI deployment with zones here.

Reply:
In Volume Topology section, we have examples for `WaitForFirstConsumer`.
Example here are for `Immediate` volume binding. Let me move this section there, and fix workflow.

Reply:
@SandeepPissay PR got merged before I can address your review comment. 

Addressed your comment in this PR - https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/176